CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Python Buildpack Changelog
 
+## 103
+
+Bug fixes.
+
+- Fix for Pipenv.
+- Fix for Heroku CI.
```

Comment:
Looking at https://github.com/heroku/heroku-buildpack-python/compare/v102...e621ff4, there's also:
- Improve handling of `WEB_CONCURRENCY` when using multiple buildpacks.
- Adjust environment variables set during the build to more closely match those in the dyno environment (`DYNO` is now available, `STACK` is not).
- Restore the build cache prior to running `bin/pre_compile`.

:-)